dev-packages/browser-integration-tests/suites/public-api/instrumentation/onError/rethrown/test.ts

```diff
@@ -24,19 +38,5 @@ sentryTest(
         frames: expect.any(Array),
       },
     });
-
-    // This is not a refernece error, but another generic error
-    expect(events[1].exception?.values).toHaveLength(1);
-    expect(events[1].exception?.values?.[0]).toMatchObject({
-      type: 'Error',
-      value: 'error 2',
-      mechanism: {
-        type: 'generic',
-        handled: true,
-      },
-      stacktrace: {
-        frames: expect.any(Array),
-      },
-    });
```

Comment:
I think this was there before to ensure that the error thrown "in between" is not sent to Sentry. Probably not a super nice way to check that but ensuring something is _not_ sent is always a bit tricky 😅 

If you have a better idea how to check for this feel free to go for it; otherwise let's bring it back, wdyt?

Reply:
I think we can switch back to manually calling `window.onerror` just for this case here. When we call this without the outer `try/catch` block, it doesn't call the second `captureException` as the page crashes.